docs/www/tls-kubernetes.md

```diff
@@ -0,0 +1,375 @@
+
+When you finish setting up your configuration, apply it by running:
+
+```bash
+kubectl apply \
+-f <your-file-path.yaml>
+```
+
+Remember to change to the actual file path that you used. 
+
+
+## Managing Certificates
+
+### Certificates Creation
+
+The Redpanda operator uses [cert-manager](https://cert-manager.io) to generate certificates. 
+
+By default, when you enable TLS, the Redpanda operator generates a root certificate for each API and uses that to generate leaf certificates for the nodes and client. You can check what certificates were generated in the section down below. 
+
+Specifically for the Kafka API, the operator also offers two options: 
+
+1) provide an existing certificate issuer; 
+   
+2) provide a node certificate. 
+
+This functionality is only available for the Kafka API.
+
+To provide an existing certificate issuer we need to fill in the `issuerRef`.
+
+In the `yaml` configuration file add this section:
+
+```yaml
+    kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+        issuerRef:
+          name: <issuer-name>
+          kind: ClusterIssuer
+```
+
+Similarly, to provide a particular certificate (as a secret) as the node certificate we need to fill in the `nodeSecretRef` field:
+
+```yaml
+    kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+        nodeSecretRef:
+          name: cluster-tls-node-certificate
+          namespace: given-cert
+```
+
+If the secret is in a namespace different from the namespace of the Redpanda cluster, the operator copies it over.
+
+### Reading/Inspecting the certificates
+
+When enabling TLS, the Redpanda operator generates a root certificate for the Kafka API that is consecutively used to generate a “node” certificate for the Kafka API listener. 
+
+If client authentication (mTLS) is enabled, an additional “client” certificate is generated. Each certificate is stored in a Kubernetes secret resource under the same namespace as the Redpanda cluster. The Certificate name is the same as that of the corresponding secret.
+
+Here’s the list of secrets for the Kafka API:
+
+Root certificate
+* Used to generate the node and client certificates
+* Secret: `<cluster>-kafka-root-certificate`
+
+Node certificate
+* The node certificate is automatically provided to Redpanda by the operator. In particular, the certificate secret is mounted as a volume that is consumed by Redpanda.
+* Secret: `<cluster>-redpanda`
+
+Client certificate (applicable only when client authentication is enabled)
+* The client certificate is available for consumption. Depending on where the client is located, the client certificate secret can be mounted to local pods or be exported out of the Kubernetes cluster, as long as the Admin API is exposed externally.
+* Secret: `<cluster>-user-client`
+* Available for client use
+
+Here’s the table of certificates, the “client” certificates are only needed and generated for client authentication (mTLS).
+
+
+<table>
```

Comment:
Not a big deal and I want to change the way tables appear in our doc anyway, but without the markdown notation the tables don't have alternating colors for rows or bold headings. I think this might look better though. And it is wrapping the lines instead of making it impossible to read? You could see how it looks if you put the top row and left column in bold.

Reply:
There's an issue with how Gatsby display tables with code fences right now. I removed the code fences so it's easier to read. 
It's possible to add HTML tags to the table to make it bold, italic and stuff like that. I'll add it for the titles.

Reply:
Sounds good